# Barry Moore

3417 S. Crestwood Dr.  
Millcreek, UT 84109  
[barry.utah@gmail.com](mailto:barry.utah@gmail.com) | (801) 243-8819 | LinkedIn: [https://www.linkedin.com/in/barrymoore66](https://www.linkedin.com/in/barrymoore66/)

## PROFESSIONAL SUMMARY

Senior Scientist and Bioinformatics Engineer with over 20 years of experience at the forefront of genomic analysis, software development, and rare disease diagnostics. Proven leader in building robust NGS processing workflows, designing clinically actionable genomic analysis pipelines, and translating complex genomic data into actionable scientific insights. Equally fluent in biology, genome informatics, and engineering, with a strong track record of impactful collaborations in both academic and commercial settings.

## CORE COMPETENCIES

* Clinical Genomics & Rare Disease Diagnostics
* NGS Processing Pipeline Development (WGS, RNASeq)
* Genomics Software Engineering (Perl, Python)
* Development of and Analysis with Disease Gene/Variant Prioritization and Genome Annotation Tools (VAAST, VVP, GEM, MAKER)
* HIPPA & PHI TCAP/CLIA Regulatory Familiarity
* NLP Based Electronic Health Record Phenotype Profiling
* HPC/Cloud Scale Workflow Automation & Big Data Integration
* Interdisciplinary Team Leadership & Trainee Mentorship
* Extensive Scientific Communication & Technical Training Experience

## EXPERIENCE
**Director of Research and Science**  
*Utah Center for Genetic Discovery, University of Utah | 2014 – Present*

* Leadership role in the analytical arm of NeoSeq and GeneKids projects focused on rapid whole genome interpretation for critically ill children.
* Developed clinical interpretation pipelines and SOPs for research-clinical integration of WGS and RNASeq data.
* Created tools like PhenVen for HPO-driven phenotype matching supporting of WGS/RNASeq diagnostics.
* Led RNASeq-based reanalysis efforts to improve diagnostic yield.
* Mentored clinicians, postdocs, graduate students, and analysts across genomics teams.

**Research Scientist**  
*Department of Human Genetics, University of Utah | 2006 – 2014*

* Designed and implemented scalable & repeatable WGS pipelines for disease gene discovery.
* On the development team multiple tools (VAAST, VVP, Phevor) adopted in genomic analysis and deployed them in analysis of dozens of large genomic datasets - large and small.
* Led bioinformatics support across collaborations with ARUP, Regeneron, Clinithink, Childhood Liver Disease Research Network and more.

**Senior Research Specialist**  
*Department of Human Genetics, University of Utah | 1994 – 2006*

* Self directed transition from molecular biology to genome informatics during the early genomic era.
* Managed lab operations, trained junior researchers, and built early genome analysis tools.
* Key member of multiple large-scale genomic research initiatives.
* Instructor in numerous CSHL bioinformatics and NGS sequencing courses.

**Senior Consultant**  
*Fabric Genomics (formerly Omicia) | 2009 – Present*

* Long-term contributor to the GEM AI platform and full gene/variant prioritization stack.
* Built and curated all major training datasets for VAAST/VVP/Phevor/GEM.
* Supported collaborations with Genomics England, Rady Children’s, GeneDx, ONT and other Fabric partners.
* Built NGS secondary (variant calling) frameworks for Fabric and key customers based on GATK, Sentieon and Dragen.

## EDUCATION & TRAINING

* M.S., Biology (Molecular Biology & Systematics), Loma Linda University, 1993
* B.S., Chemistry, Southwestern Adventist University, 1990
* Genome Informatics, Cold Spring Harbor Laboratory, 2003
* Summer Institute in Statistical Genetics, University of Washington, 2010

## SELECTED PUBLICATIONS

* Reese et al., *Genome Biology*, 2010: Standard Variation File Format
* Yandell et al., *PLoS Comp Biol*, 2008: Disease Alleles in Paralogous Proteins
* Eilbeck et al., *BMC Bioinformatics*, 2009: Quantitative Measures for Genome Annotation
* Moore et al., *Genet Med*, 2011: Variation in 10 Healthy Genomes and Diagnostic Implications
* Hu et al., *Genet Epidemiol*, 2013: VAAST 2.0 Variant Classification
* Flygare et al., *BMC Bioinformatics*, 2018: VAAST Variant Prioritizer (VVP)
* De La Vega et al., *Genome Med*, 2021: AI-Enabled Genome Interpretation
* Moore et al., *Genet Med Open*, 2023: RNASeq Resolves NEB Variant in Nemaline Myopathy
* Jenkins et al., *NPJ Genom Med*, 2025: Utah NeoSeq Program in NICU Genomic Diagnostics

*Full list of 76 publications available on request or Google Scholar - [https://bit.ly/3IKbfLV](https://bit.ly/3IKbfLV)*</br>
*Citation metrics (Google Scholar, July 2025): h-index = 31, i10-index = 41, total citations = 6,522*

## TOOLS & TECHNOLOGIES

* **Languages:** Perl, Python, JavaScript, SQL
* **Linux Admin & Power Tools:** Bash, screen/tmux, git, emacs, cron/at, find, parallel/xargs, sed/awk/CLI Perl, VisiData, jq, sqlite3, rclone/rsync
* **Workflow Systems:** Nextflow, Snakemake, custom scripting
* **Cloud/Infra:** AWS, Docker, Linux HPC clusters
* **Bioinformatics Tools:** GATK, Sentieon, Dragen, samtools, bcftools, bedtools, htslib, vcfanno, GEM, VAAST, VVP, Phevor
* **Data Analysis/Presentation:** iPython/Jupyter, numpy, Pandas, scipy, matplotlib/Seaborn/Plotly, custom dashboards & worksheets using JavaScript, DataTables, HTML/CSS, XlsxWriter, Markdown

## PATENTS

US Patent Application No. US20230326547A1 **Systems and Methods for Prioritizing Genome Variants.** Filed October 2023. ***Co-inventor***. [https://patents.google.com/patent/US20230326547A1](https://patents.google.com/patent/US20230326547A1)

_References available upon request._
