---
title: "Barry Moore, M.S."
layout: home
---

# Barry Moore, M.S.

**Senior Scientist, Precision Genomics**  
*Genome Scientist | Bioinformatics Builder | Big Data Freak*</br>
[LinkedIn](https://www.linkedin.com/in/barrymoore66/) | [GitHub](https://github.com/barrymoore) | <a href="mailto:barry.utah&#64;gmail.com">barry.utah&#64;gmail.com</a>

---

## About Me
I’m a genome scientist, bioinformatics builder and big data evangelist with over 25 years of experience at the intersection of biology, computation, and clinical translation. My career began at the bench, decoding the complexities of translational recoding, and has evolved into a deep engagement with genome-scale data and rare disease diagnostics.

At the University of Utah, I’ve helped lead some of the institution’s most impactful genomic projects — from working on the development team for software tools like MAKER, VAAST, and VVP to shaping diagnostic workflows for programs like Heritage 1K, NeoSeq and GeneKids. My work spans clinical genome diagnostics and NLP-base phenotype profiling, genome annotation and comparative genomics, ontology design, and consortia-scale genomic and data science workflows.

As a long-time consultant for Fabric Genomics, I’ve contributed to the design and optimization of their AI-driven genome interpretation engine, GEM. Over more than a decade, I’ve worked with Fabric to develop robust bioinformatics workflows, refine variant and disease gene prioritization strategies, and lead analysis efforts on high impact research collaborations related to diagnosis of rare disease.

Throughout my career, I’ve focused on long-term trust, collaboration with cross-disciplinary fluency, and delivering work that matters. I strive to be the person that teams call on when something needs to work — bridging biology, engineering, and clinical care.

I am always curious about what’s next — in genetics, genomics and technology and how we turn advances in these fields into tools for impact.

---

## Areas of Expertise
* Clinical Genomics & Rare Disease Diagnostics
* NGS Processing Pipeline Development (WGS, RNASeq)
* Genomics Software Engineering (Python, Perl)
* Development of and Analysis with Disease Gene/Variant Prioritization and Genome Annotation Tools (VAAST, VVP, GEM, MAKER)
* HIPPA/PHI Training
* NLP Based Electronic Health Record Phenotype Profiling
* HPC/Cloud Scale Workflow Automation & Big Data Integration
* Interdisciplinary Team Leadership & Trainee Mentorship
* Extensive Scientific Communication & Technical Training Experience

---

## Career Highlights

**Director of Research and Science**  
*Utah Center for Genetic Discovery, University of Utah | 2014 – Present*
* Leadership role in the analytical arm of NeoSeq and GeneKids projects focused on rapid whole genome interpretation for critically ill children.
* Developed clinical interpretation pipelines and SOPs for research-clinical integration of WGS and RNASeq data.
* Created tools like PhenVen for HPO-driven phenotype matching supporting of WGS/RNASeq diagnostics.
* Led RNASeq-based reanalysis efforts to improve diagnostic yield.
* Mentored clinicians, postdocs, graduate students, and analysts across genomics teams.

**Research Scientist**  
*Department of Human Genetics, University of Utah | 2006 – 2014*
* Designed and implemented scalable & repeatable WGS pipelines for disease gene discovery.
* On the development team multiple tools (VAAST, VVP, Phevor) adopted in genomic analysis and deployed them in analysis of dozens of large genomic datasets - large and small.
* Led bioinformatics support across collaborations with ARUP, Regeneron, Clinithink, Childhood Liver Disease Research Network and more.

**Senior Research Specialist**  
*Department of Human Genetics, University of Utah | 1994 – 2006*
* Self directed transition from molecular biology to genome informatics during the early genomic era.
* Managed lab operations, trained junior researchers, and built early genome analysis tools.
* Key member of multiple large-scale genomic research initiatives.
* Instructor in numerous CSHL bioinformatics and NGS sequencing courses.

**Senior Consultant**  
*Fabric Genomics (formerly Omicia) | 2009 – Present*
* Long-term contributor to the GEM AI platform and full gene/variant prioritization stack.
* Built and curated all major training datasets for VAAST/VVP/Phevor/GEM.
* Supported collaborations with Genomics England, Rady Children’s, GeneDx, ONT and other Fabric partners.
* Built NGS secondary (variant calling) frameworks for Fabric and key customers based on GATK, Sentieon and Dragen.

---

## Selected Projects
- **NeoSeq** – Rapid genome sequencing for NICU diagnostics at University of Utah Health
- **GEM AI** – Development and clinical integration of AI-assisted genome interpretation at Fabric Genomics
- **GeneKids** – Rare disease diagnostics initiative at Intermountain Health and University of Utah
- **Children's Liver Disease Research Network** - A collaborative team of clincians and scientists supported by the NIH to study a broad range of childhood liver disease.
- **Sequence Ontology** - The SO is a collaborative ontology project for the definition of sequence features used in biological sequence annotation.

---

## Selected Publications
* Reese et al., *Genome Biology*, 2010: Standard Variation File Format
* Yandell et al., *PLoS Comp Biol*, 2008: Disease Alleles in Paralogous Proteins
* Eilbeck et al., *BMC Bioinformatics*, 2009: Quantitative Measures for Genome Annotation
* Moore et al., *Genet Med*, 2011: Variation in 10 Healthy Genomes and Diagnostic Implications
* Hu et al., *Genet Epidemiol*, 2013: VAAST 2.0 Variant Classification
* Flygare et al., *BMC Bioinformatics*, 2018: VAAST Variant Prioritizer (VVP)
* De La Vega et al., *Genome Med*, 2021: AI-Enabled Genome Interpretation
* Moore et al., *Genet Med Open*, 2023: RNASeq Resolves NEB Variant in Nemaline Myopathy
* Jenkins et al., *NPJ Genom Med*, 2025: Utah NeoSeq Program in NICU Genomic Diagnostics

*Full list of 76 publications available on request or Google Scholar - [https://bit.ly/3IKbfLV](https://bit.ly/3IKbfLV)*</br>
*Citation metrics (Google Scholar, July 2025): h-index = 31, i10-index = 41, total citations = 6,522*

---

## Tools & Technologies
* **Languages:** Perl, Python, JavaScript, SQL
* **Linux Admin & Power Tools:** Bash, screen/tmux, git, emacs, cron/at, find, parallel/xargs, sed/awk/CLI Perl, VisiData, jq, sqlite3, rclone/rsync
* **Workflow Systems:** Nextflow, Snakemake, custom scripting
* **Cloud/Infra:** AWS, Docker, Linux HPC clusters
* **Bioinformatics Tools:** GATK, Sentieon, Dragen, samtools, bcftools, bedtools, htslib, vcfanno, GEM, VAAST, VVP, Phevor
* **Data Analysis/Presentation:** iPython/Jupyter, numpy, Pandas, scipy, matplotlib/Seaborn/Plotly, custom dashboards & worksheets using JavaScript, DataTables, HTML/CSS, XlsxWriter, Markdown

## Patents

US Patent Application No. US20230326547A1Systems and Methods for Prioritizing Genome Variants. Filed October 2023. Co-inventor. [https://patents.google.com/patent/US20230326547A1](https://patents.google.com/patent/US20230326547A1)

---

**barrymoore.bio** — All views are my own. Thanks for visiting.

